go.mod

```diff
@@ -43,6 +45,7 @@ require (
 	github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4 // indirect
 	github.com/99designs/keyring v1.2.1 // indirect
 	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d // indirect
+	github.com/CosmWasm/wasmvm v1.5.2 // indirect
```

Comment:
Could this be wrong?

See this compatibility table:
https://github.com/CosmWasm/wasmd?tab=readme-ov-file#for-node-developers

Reply:
Good thought! But I believe the versions should be compatible. Stargaze and Juno both use this combo

I think the way to read that table is that wasmd `v0.45.0` uses wasmvm `v1.5.0` in it's own go.mod, _but_ wasmd `v0.45.0` should be compatible with any `v1.5.x`

Fwiw, I tried a few combinations of versions (I think I tried v1.5.0, v1.5.1 and v1.5.2) and got the same error with all of them 😞